rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -147,11 +144,11 @@ fi
 # http://www.rudder-project.org/redmine/issues/8065
 if [ ${CFRUDDER_FIRST_INSTALL} -eq 1 ]
 then
-  if type setfacl > /dev/null 2> /dev/null; then
+  if type setfacl >> ${LOG_FILE} 2>&1; then
     setfacl -R -k /var/rudder/
   fi
 fi
 
 # launch rudder agent check script, it will generate an UUID on first install or repair it if needed
-nohup /opt/rudder/bin/check-rudder-agent >/dev/null 2>/dev/null &
+${RUDDER_CMD} agent check >> ${LOG_FILE} 1>&1
```

Comment:
Because it is not needed (agent check is quick) and if the check fails, i means the agent has failed to install properly, so we should keep its result in the postinstall